partner_credit_limit_history/wizard/partner_credit_limit_wizard.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class PartnerCreditLimitWizard(models.TransientModel):
+    _name = "partner.credit.limit.wizard"
+    _description = "Partner credit limit wizard"
+
+    partner_id = fields.Many2one(
+        comodel_name="res.partner", string="Partner", required=True
+    )
+    amount = fields.Float(string="New Amount", required=True)
+    note = fields.Char(string="Comments", required=True)
+
+    def action_confirm(self):
+        history = (
+            self.env["res.partner.credit.history"]
+            .sudo()
```

Comment:
Maybe add a comment here explaining why the need of `sudo` for future reference, as explained in https://github.com/OCA/credit-control/pull/142#discussion_r682814841